src/components/common/target-list-item.scss

```diff
@@ -1,41 +1,54 @@
 @import '../../style/common/constants';
 
 .target-item {
-  width: 25vw;
+  width: 26vw;
   display: flex;
-  padding-right: 0;
-  margin-right: 0;
-  margin-left: 2em;
-  padding-bottom: 0;
 }
 
 .target-item:hover {
   border: 4px solid $dodger-blue;
 }
 
+.target-item__trash-button-container {
+  text-align: left;
+  padding-top: 0.2em;
+  height: 2em;
+}
+
+.target-item__trash-button-img {
+  width: 30px;
+}
+
+.target-item__trash-button:hover {
+  background-color: $alizarin-crimson;
+}
+
 .target-item__text-container {
-  align-self: left;
+  text-align: left;
   flex-grow: 2;
+  margin-left: 2vw;
+  padding-left: 1em;
```

Comment:
I would use rem for setting padding and margins since it makes easier to identify what the actual value of the unit is along your project, using em could make it harder because 1em here could be different from 1em in other place(will take the value of the font-size from the current element, while rem will take  the value from the root html element which you should already define previously).

Reply:
Ok I'll implement this in another PR